lib/helpers.d.ts

```diff
@@ -0,0 +1,18 @@
+export declare function isTouch(): boolean;
+export declare function isTouchEvent(event: any): event is TouchEvent;
```

Comment:
is it not suppose to be export declare function isTouchEvent(event: any): Boolean?

Reply:
This is an enhanced Boolean actually
The [TS Type guard
](https://www.typescriptlang.org/docs/handbook/advanced-types.html?#user-defined-type-guards)